wp-graphql-buddypress.php

```diff
@@ -226,6 +226,26 @@ private function filters() {
 			 * Setup filters
 			 */
 			\WPGraphQL\Extensions\BuddyPress\TypeRegistry::add_filters();
+
+			/**
```

Comment:
By default, wp-graphql blocks the visibility of users without posts published. BuddyPress has a more open approach, as every member is public via the their profile.

This change allows the API to show users publicly without exposing sensitive information.